android/src/com/mapswithme/maps/search/SearchAdapter.java

```diff
@@ -160,25 +161,40 @@ private CharSequence formatDescription(SearchResult result)
       final SpannableStringBuilder tail = new SpannableStringBuilder();
 
       final int stars = Math.min(result.description.stars, 5);
-      if (stars > 0)
+      if (stars > 0 || !result.description.rating.isEmpty())
       {
-        // Colorize last dimmed stars
-        final SpannableStringBuilder sb = new SpannableStringBuilder("★ ★ ★ ★ ★");
-        if (stars < 5)
+        if (stars > 0)
         {
-          final int start = sb.length() - ((5 - stars) * 2 - 1);
-          sb.setSpan(new ForegroundColorSpan(itemView.getResources().getColor(R.color.search_star_dimmed)),
-                     start, sb.length(), Spanned.SPAN_INCLUSIVE_EXCLUSIVE);
+          // Colorize last dimmed stars
+          final SpannableStringBuilder sb = new SpannableStringBuilder("★ ★ ★ ★ ★");
+          if (stars < 5)
```

Comment:
Can't imagine more complex way to create a string of |stars| stars separated by spaces. Simplify, please.